app/src/main/res/values/strings.xml

```diff
@@ -733,7 +733,7 @@
     <string name="on">On</string>
     <string name="off">Off</string>
     <!-- Progressive Load Interval -->
-    <string name="progressive_load_interval_default">ExoPlayer default</string>
+    <string name="progressive_load_interval_exoplayer_default">ExoPlayer default</string>
```

Comment:
Shouldn't the corresponding string in other languages be renamed too?

Reply:
Yes ;-)